File: Sep/Day4/find-closest-person.java
//  Time: O(1)  ::  Space: O(1)
class Solution {
    public int findClosest(int x, int y, int z) {
        int personOneDiff = Math.abs(z - x);
        int personTwoDiff = Math.abs(z - y);
        if (personOneDiff == personTwoDiff) {
            return 0;
        }
        return (personOneDiff < personTwoDiff) ? 1 : 2;
    }
}

File: Sep/Day4/find-closest-person.py
#  Time: O(1)  ::  Space: O(1)
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        personOneDiff = abs(z-x)
        personTwoDiff = abs(z-y)
        if personOneDiff == personTwoDiff:
            return 0
        return 1 if personOneDiff < personTwoDiff else 2